### PR TITLE
Reality Kings site change

### PR DESCRIPTION
**Pull request by SgtBatten**

### CODE REVIEW

**Comment by d2dyno** [comment]
Tested on recent scenes from BigNaturals, RkPrime and RoundAndBrown without issues. Thanks for the quick update :+1:

File: docs/sitelist.md [hunk 1]
```diff
@@ -585,7 +585,7 @@ The above is a reference list. Please see [the manualsearch doc](./manualsearch.
   - Tight and Teen
 + #### PropertySex | Matching type: *[Exact](./manualsearch.md#exact-match)* - **SceneID**
 + #### PureTaboo | Matching type: *[Enhanced](./manualsearch.md#enhanced-search)*
-+ #### Reality Kings Network | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### Reality Kings Network | Matching type: *[Limited](./manualsearch.md#exact-match)* - **SceneID**
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Could you change the text to "Exact"? Everything else looks good to me!

File: Contents/Code/siteRealityKings.py [hunk 2]
```diff
@@ -1,147 +1,126 @@
...
+        sceneTitle = encodedTitle.split('%20', 1)[1].replace('%20',' ')
+    except:
+        sceneTitle = ''
+    Log("Scene Title: " + sceneTitle)
+    url = PAsearchSites.getSearchSearchURL(siteNum) + sceneID + "/1"
+    searchResults = HTML.ElementFromURL(url)
+    for searchResult in searchResults.xpath('//h1'):
+        titleNoFormatting = searchResult.xpath('//h1')[0].text_content().replace('Trailer','').strip()
+        curID = url.replace('/','_').replace('?','!')
+        subSite = searchResult.xpath('//div[@class="sc-11m21lp-2 fOadtn"]')[0].text_content().strip()
```
Comment (kamuk90) [inline Contents/Code/siteRealityKings.py]: this is didn't work

**Comment by Keysersose12** [comment]
Is this working? If so can someone give an example search name as its not working for me.

**Pull request merged**